tests/fixtures/datetime/expected/Normalizer/TestNormalizer.php

```diff
@@ -42,6 +42,29 @@ public function denormalize($data, $class, $format = null, array $context = [])
         if (property_exists($data, 'date')) {
             $object->setDate(\DateTime::createFromFormat("Y-m-d\TH:i:sP", $data->{'date'}));
         }
+        if (property_exists($data, 'dateOrNull')) {
+            $value = $data->{'dateOrNull'};
+            if (is_object($data->{'dateOrNull'}) and false !== \DateTime::createFromFormat("Y-m-d\TH:i:sP", $data->{'dateOrNull'})) {
```

Comment:
I wonder why the is_object check is there. This is what I fixed in #30. At this point the data is not an object, but a string.